content/tsColinPrecip.Rmd

```diff
@@ -86,28 +84,22 @@ precipMap <- function(precipData, startDate, endDate){
   precip_breaks <- c(seq(0,80,by = 10), 200)
   
   precipData_cols <- precipData %>% 
-    group_by(state_fullname, county_mapname) %>% 
+    group_by(statename, statecounty) %>% 
```

Comment:
I think you mean fips and yes you could, but I need to use the state and county names to make the map, so I grouped by these in order to keep those in the data frame.